src/framework/preferences/css/stylus/PrefsEditor.styl

```diff
@@ -1,8 +1,8 @@
 /* General Preferences Editor styles, used for all layouts */
+// Deprecated function call to contrastThemes and fontFamily styling
```

Comment:
The depreciation comments should actually be in Themes.styl, also don't need to mention fontFamily. Similar for other depreciation comments throughout.